corehq/apps/app_manager/templates/app_manager/partials/forms/usercase_config.html

```diff
@@ -8,12 +8,6 @@
 </script>
 
 <div id="usercase-config-ko">
-  <div class="clearfix">
-    {% if allow_usercase %}
-      <div class="refresh-form-questions-container" data-bind="template: 'case-config:refresh-form-questions'"></div>
-      <div data-bind="saveButton: saveUsercaseButton"></div>
```

Comment:
I think this deletes the "Save" button inside the User Properties tab, not just the refresh button.

Reply:
Indeed, thanks. Fixed in 06831039260